tests/test_metrics.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+
+import numpy as np
+from scipy.stats import ortho_group
+from sklearn.datasets import load_iris
+
+from skcosmo.metrics import (
+    global_reconstruction_error,
+    global_reconstruction_distortion,
+    local_reconstruction_error,
+)
+
+
+class ReconstructionMeasuresTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        features = load_iris().data
+        cls.features_small = features[:20, [0, 1]]
+        cls.features_large = features[:20, [0, 1, 0, 1]]
+        cls.eps = 1e-5
+        cls.n_local_points = 15
+
+        np.random.seed(0x5F3759DF)
+        cls.features_rotated_small = cls.features_small.dot(
+            ortho_group.rvs(cls.features_small.shape[1])
+        )
+
+    def test_global_reconstruction_error_identity(self):
+        gfre_val = global_reconstruction_error(self.features_large, self.features_large)
+        self.assertTrue(
+            abs(gfre_val) < self.eps,
+            f"global_reconstruction_error {gfre_val} surpasses threshold for zero {self.eps}",
+        )
+
+    def test_global_reconstruction_error_small_to_large(self):
```

Comment:
Shouldn't small --> large be a larger GRE not equation to 0?

```suggestion
    def test_global_reconstruction_error_small_to_large(self):
    # tests that the GRE of a small dataset onto a larger dataset returns within a threshold of zero
```

Reply:
you mean: why it should be zero? Because large features was set such that it can be linearly reconstructed from small features, if you look at the definition of large feature it becomes clear.